java/configuration/src/main/java/sleeper/configuration/properties/instance/UserDefinedInstancePropertyImpl.java

```diff
@@ -154,6 +163,11 @@ public Builder propertyGroup(PropertyGroup propertyGroup) {
             return this;
         }
 
+        public Builder defafultProperty(SleeperProperty property) {
```

Comment:
There's a typo at `defafultProperty`.

I also think it would be more consistent to replace the parameter name `property` with `defaultProperty`.

Reply:
Addressed